project_template/models.py

```diff
@@ -72,6 +72,8 @@ def __str__(self):
         )
 
     def add_position(self, position):
+        if self.all_positions is None:
```

Comment:
I would rather this check read `if not all_positions`, since `is` checks for identity (whereas `==` checks for equality). 

Since there is no valid/desirable state where `self.all_positions` would be an empty list, we can safely assume that, if it is falsey, it needs to be initiated with an empty list.